src/components/Modal/Modal.tsx

```diff
@@ -333,6 +339,10 @@ class Modal extends React.Component<CombinedProps, State> {
     if (this.focusReturnPointNode) {
       write(() => focusFirstFocusableNode(this.focusReturnPointNode, false));
     }
+
+    this.activatorRef &&
+      this.activatorRef.current &&
+      focusFirstFocusableNode(this.activatorRef.current);
```

Comment:
We should be able to drop `this.activatorRef &&` since we initialize  activatorRef with `createRef` which is the "sort of equivalent" of `private activatorRef = {current: undefined}`